### PR TITLE
Update deploying-to-azure-kubernetes-service.md

### PR DESCRIPTION
**Pull request by leonardoacosta**
### Why:
Extra '-', this causes the above name to look like it has no `use` or `run` step.
Also Bumped the versions of all the actions used

Closes: 

N/A

### What's being changed (if available, include any code snippets, screenshots, or gifs):

N/A

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ x ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by leonardoacosta** [comment]
> @leonardoacosta Thanks for the PR! I'll get it up for review ⚡
> 
> In the mean time, would you mind expanding a bit more on your change to line 115? Updating `AZURE_CONTAINER_REGISTRY`

 Thank you, yea!

`AZURE_CONTAINER_REGISTRY` should look like: `contoso`, while `REGISTRY_URL` looks like `contoso.azurecr.io`

Line 115 used to look like this:
```
 {% raw %}${{ env.AZURE_CONTAINER_REGISTRY }}{% endraw %}.azurecr.io/{% raw %}${{ env.PROJECT_NAME }}{% endraw %}:{% raw %}${{ github.sha }}{% endraw %}
 ```
 
 Where there's the extra '.azurecr.io' so I thought why not just use the variable with the full URL, `REGISTRY_URL`:
 
 Update:
 ```
 {% raw %}${{ env.REGISTRY_URL }}{% endraw %}/{% raw %}${{ env.PROJECT_NAME }}{% endraw %}:{% raw %}${{ github.sha }}{% endraw %}
 ```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md [hunk 1]
```diff
@@ -76,28 +76,28 @@ jobs:
     - uses: {% data reusables.actions.action-checkout %}
 
     - name: Azure Login
-      uses: azure/login@14a755a4e2fd6dff25794233def4f2cf3f866955
+      uses: azure/login@v1
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md:79]: (here and elsewhere) Our style guide requires us to use the full SHA for third party actions. It's ok if you want to update the workflow to use a later version, but we'll need to specify the SHA that corresponds to the new verison.

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md [hunk 2]
```diff
@@ -108,11 +108,11 @@ jobs:
       id: bake
 
     - name: Deploys application
-    - uses: Azure/k8s-deploy@dd4bbd13a5abd2fc9ca8bdcb8aee152bb718fa78
+      uses: Azure/k8s-deploy@v4
       with:
         manifests: {% raw %}${{ steps.bake.outputs.manifestsBundle }}{% endraw %}
         images: |
-          {% raw %}${{ env.AZURE_CONTAINER_REGISTRY }}{% endraw %}.azurecr.io/{% raw %}${{ env.PROJECT_NAME }}{% endraw %}:{% raw %}${{ github.sha }}{% endraw %}
+          {% raw %}${{ env.REGISTRY_URL }}{% endraw %}/{% raw %}${{ env.PROJECT_NAME }}{% endraw %}:{% raw %}${{ github.sha }}{% endraw %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md:115]: Can you tell us more about why you made this change?

**Review by skedwards88 — COMMENTED** [review]
Thanks for this PR! Just a couple of requests before we can merge this.

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md (hunk 2 above)
Comment (skedwards88) [reply content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md:115]: I see your comment in https://github.com/github/docs/pull/28563#issuecomment-1739330387 now

**Pull request closed without merging**